docs/sources/setup-grafana/configure-security/configure-authentication/azuread/index.md

```diff
@@ -201,16 +243,46 @@ allowed_organizations = 8bab1c86-8fba-33e5-2089-1d1c80ec267d
 
 ### Configure allowed groups
 
-To limit access to authenticated users who are members of one or more groups, set `allowed_groups`
-to a comma- or space-separated list of group object IDs. You can find object IDs for a specific group on the Azure portal:
+Azure AD groups can be used to limit user access to Grafana. For more information about managing groups in Azure AD, refer to [Manage Microsoft Entra groups and group membership](https://learn.microsoft.com/en-us/entra/fundamentals/how-to-manage-groups).
+
+To limit access to authenticated users who are members of one or more AzureAD groups, set `allowed_groups`
+to a comma- or space-separated list of group object IDs.
+
+Complete the following step to find object IDs for a specific group on the Azure portal:
 
-1. Go to **Azure Active Directory -> Groups**. If you want to only give access to members of the group `example` with an ID of `8bab1c86-8fba-33e5-2089-1d1c80ec267d`, then set the following:
+1. Go to **Azure Active Directory > Groups**.
+
+   If you want to only give access to members of the group `example` with an Object Id of `8bab1c86-8fba-33e5-2089-1d1c80ec267d`, then set the following:
```

Comment:
Ths section with the example feels out of place here.  I would add this as an example at the end of the instructions.

Reply:
Moved it after the 'You can find the Object Id...' note.